4337/README.md

```diff
@@ -4,7 +4,7 @@
 
 ## Execution Flow
 
-The diagram below outlines the flow that is triggered when a user operation is submitted to the entrypoint. Additionally the gas overhead compared to a native implementation is mentioned.
+The diagram below outlines the flow triggered when a user operation is submitted to the entrypoint. Additionally, the gas overhead compared to a native implementation is mentioned.
```

Comment:
Interesting, the sentence feels weird without the "that is", but I trust Grammarly more than me 😅.